src/utils/ng-validate.ts

```diff
@@ -4,13 +4,15 @@ import {MessageType} from '../communication/message-type';
 import {send} from '../backend/indirect-connection';
 
 declare const getAllAngularTestabilities: Function;
+declare const getAllAngularRootElements: Function;
+declare const ng: any;
 
 let unsubscribe: () => void;
 
 const handler = () => {
   // variable getAllAngularTestabilities will be defined by Angular
```

Comment:
@stevenkampen seems like you are not using getAllAngularTestabilities anymore. Should this comment be updated? Also, how would that affect previous version support?